sig-windows/README.md

```diff
@@ -92,4 +92,16 @@ As always, you can also find the active list of open issues and proposals [Open
 
 Past meetings are in the [Recorded Meetings Playlist on Youtube](https://www.youtube.com/playlist?list=PL69nYSiGNLP2OH9InCcNkWNu2bl-gmIU4&jct=LZ9EIvD4DGrhr2h4r0ItaBmco7gTgw)
 
+## Sig-Windows is Growing! 
+
+Sig-Windows is looking to build new commiters and owners, with the goal of eventually having new leaders and chairs.
```

Comment:
This makes it sound like some or all of us are looking for a replacements to give us an out.
I'm trying to think of how this could be reworded but don't have a great suggestion at the moment.

Reply:
ill soften the language that way we can iterate if we come up with something more realistic

Reply:
updated